extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -504,12 +507,12 @@ export class AmpLightboxViewer extends AMP.BaseElement {
 
   /**
    * Toggle description box if it has text content
-   * @param {boolean} display
    * @private
    */
-  toggleDescriptionBox_(display) {
+  showDescriptionBox_() {
+    this.descriptionBox_.classList.remove('fade-out');
     this.updateDescriptionBox_();
-    toggle(dev().assertElement(this.descriptionBox_), display);
+    toggle(dev().assertElement(this.descriptionBox_), true);
```

Comment:
there should not be a need to keep `toggle` given opacity/visibility in CSS will handle it.